src/odemis/gui/cont/tabs/sparc2_align_tab.py

```
@@ -833,6 +864,85 @@ def add_axis(axisname, comp, label=None):
         # TODO: Auto remove the background when the image shape changes?
         # TODO: Use a toggle button to show the background is in use or not?
 
+    def _on_btn_auto_align(self, evt):
```

Ruff (0.14.13)

Unused method argument: `evt`

(ARG002)